### PR TITLE
Suppress RuntimeError in EE websocket

### PR DESCRIPTION
**Pull request by jonathan-eq**
**Issue**
Connected to #7642 

**Approach**
The commit in this PR should fix the issue where the websocket server would raise RuntimeError stating `no running event loop` under heavy load. The fix here is to wrap it in a try-except, and checking if the error message contains `no running event loop`.

(Screenshot of new behavior in GUI if applicable)

- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Make sure tests pass locally (after every commit!)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

**Comment by jonathan-eq** [comment]
This was the suggested fix by chatgpt. We could also pass in our own logger and filter messages instead, but that would be more messy.

**Comment by berland** [comment]
> It seems to be when we are exiting the websocket context, and it is trying to close the connections. At this point, all communication should be done; so we can ignore this failure. Are all realisations already finished when we get this error?

It was in between iterations, so the realizations for one ensemble was probably finished yes.

**Comment by jonathan-eq** [comment]
> > It seems to be when we are exiting the websocket context, and it is trying to close the connections. At this point, all communication should be done; so we can ignore this failure. Are all realisations already finished when we get this error?
> 
> It was in between iterations, so the realizations for one ensemble was probably finished yes.

Every instance of the ensemble_evaluator creates its own event loop, so they should be independent. Maybe we just get unlucky every now and then?

File: src/ert/ensemble_evaluator/evaluator.py [hunk 1]
```diff
@@ -331,59 +331,65 @@ async def process_request(
...
+                    extra_attrs=terminated_attrs,
+                    data_marshaller=cloudpickle.dumps,
                 )
+                if self._clients:
+                    # See note about return_exceptions=True above
+                    await asyncio.gather(
+                        *[client.send(message) for client in self._clients],
+                        return_exceptions=True,
+                    )
+        except RuntimeError as exc:
```
Comment (xjules) [inline src/ert/ensemble_evaluator/evaluator.py:388]: Can we check that self._done before we return?

**Comment by berland** [comment]
I think I am uncomfortable with just muting this error like this, we don't know what else we are hiding.

**Comment by jonathan-eq** [comment]
> I think I am uncomfortable with just muting this error like this, we don't know what else we are hiding.

Yes, we should fix the underlying problem instead.

**Pull request closed without merging**